ui/app/src/modules/Preview/EmbeddedLegacyEditors.tsx

```diff
@@ -124,9 +124,10 @@ export default function EmbeddedLegacyEditors(props: EmbeddedLegacyEditorsProps)
     filter((e: any) => e.data && e.data.type)
   );
 
-  const handleClose = () => {
-    setDialogConfig({ open: false, src: null, type: null, inProgress: true });
-  };
+  const handleClose = useCallback(
+    () => {
+      setDialogConfig({ open: false, src: null, type: null, inProgress: true });
+    }, [setDialogConfig]);
```

Comment:
Please reformat this and the others in this way...

```
 const handleClose = useCallback(() => {
  setDialogConfig({ open: false, src: null, type: null, inProgress: true });
}, [setDialogConfig]);
```

Reply:
Done.